Automattic/wp-calypso
# AI Logo Generator: Implement prompt enhancement functionality

Related to https://github.com/Automattic/jetpack/issues/34378
Closes https://github.com/Automattic/wp-calypso/issues/85980


## Proposed Changes

* Implements the prompt enhancement functionality within the store
* Adds the initial prompt treatment
* Changes the revised prompt to the original prompt on the image description

## Testing Instructions



* Go to the generator modal
* Type a description
* Click on "Enhance prompt"
* Check that enhancements are added
* Generate the logo
* Check that the enhanced prompt is used as the image description

## Pre-merge Checklist
